workers/gossip-server/src/peer_discovery/peers.rs

```diff
@@ -102,11 +109,19 @@ impl GossipProtocolExecutor {
                 .collect_vec()
         }; // locked_expiry_cache released
 
+        let my_cluster_id = self.global_state.get_cluster_id()?;
+        let num_local_peers =
+            filtered_peers.iter().filter(|peer| peer.cluster_id == my_cluster_id).count();
+        let num_remote_peers = filtered_peers.len() - num_local_peers;
+
```

Comment:
nit: move this above the location it's used, no reason to put unrelated lines between

Reply:
ah, this was to avoid cloning `filtered_peers` since it is moved in the call to `self.global_state.add_peer_batch`. it's likely an inconsequential clone (and maybe the compiler is smart enough to avoid it) so i'll just invoke it and move these lines